lib/config/config.go

```diff
@@ -125,17 +125,18 @@ func ReadJSON(r io.Reader, myID protocol.DeviceID) (Configuration, error) {
 }
 
 type Configuration struct {
-	Version        int                   `xml:"version,attr" json:"version"`
-	Folders        []FolderConfiguration `xml:"folder" json:"folders"`
-	Devices        []DeviceConfiguration `xml:"device" json:"devices"`
-	GUI            GUIConfiguration      `xml:"gui" json:"gui"`
-	Options        OptionsConfiguration  `xml:"options" json:"options"`
-	IgnoredDevices []ObservedDevice      `xml:"remoteIgnoredDevice" json:"remoteIgnoredDevices"`
-	PendingDevices []ObservedDevice      `xml:"pendingDevice" json:"pendingDevices"`
-	XMLName        xml.Name              `xml:"configuration" json:"-"`
-
-	MyID            protocol.DeviceID `xml:"-" json:"-"` // Provided by the instantiator.
-	OriginalVersion int               `xml:"-" json:"-"` // The version we read from disk, before any conversion
+	Version           int                   `xml:"version,attr" json:"version"`
+	Folders           []FolderConfiguration `xml:"folder" json:"folders"`
+	Devices           []DeviceConfiguration `xml:"device" json:"devices"`
+	GUI               GUIConfiguration      `xml:"gui" json:"gui"`
+	LDAPConfiguration LDAPConfiguration     `xml:"ldapConfiguration" json:"ldapConfiguration"`
```

Comment:
Drop the `Configuration` part of the JSON/XML names I think.

Reply:
Also, potentially for the member name itself? GUIConfiguration is called GUI.

Reply:
Aye